code/create_app.py

```diff
@@ -387,13 +383,28 @@ async def conversation_custom():
 
         except Exception as e:
             error_message = str(e)
-            logger.exception(
-                "Exception in /api/conversation/custom | %s", error_message
+            logger.exception("Exception in /api/conversation | %s", error_message)
+            return (
+                jsonify(
+                    {
+                        "error": "Exception in /api/conversation. See log for more details."
+                    }
+                ),
+                500,
             )
+
+    @app.route("/api/conversation", methods=["POST"])
+    async def conversation():
+        conversation_flow = env_helper.CONVERSATION_FLOW
+        if conversation_flow == "custom":
```

Comment:
Could this be better as an ENUM? This would allow us to describe what these flows do and are used for